tests/pytests/functional/formulas/test_docker.py

```diff
@@ -12,12 +12,17 @@ def _formula(saltstack_formula):
 
 @pytest.fixture(scope="module")
 def modules(loaders, _formula):
+    loaders.opts["file_roots"]["base"].append(
+        str(_formula.state_tree_path / f"{_formula.name}-{_formula.tag}")
+    )
     return loaders.modules
 
 
 @pytest.mark.skip_on_windows
 @pytest.mark.destructive_test
 def test_docker_formula(modules):
     ret = modules.state.sls("docker")
+    assert not ret.errors
+    assert not ret.failed
     for staterun in ret:
-        assert not staterun.result.failed
```

Comment:
In my testing, I found different results between `assert not staterun.result.failed` and `assert staterun.result`, and the latter was not reliable.

Reply:
I have been unable to duplicate since commenting. Please disregard.